services/comedian/hashtag_similarity.py

```diff
@@ -34,7 +90,12 @@ def get_deletable_ids(self):
     def get_clusters(self):
         d0 = {}
         for k, vSim in self.hash_groups.iteritems():
-            if len(vSim['similar_post_ids']) >= self.min_post:
+            n_terms = len(vSim['similar_post_ids'])
+            if n_terms >= self.min_post:
+                vSim['stats']['total_posts'] = self.total_posts
+                lam = float(n_terms)/self.total_posts
+                vSim['stats']['likelihood'] = gdtr(vSim['stats']['prior_beta'], vSim['stats']['prior_alpha'], lam)
+                vSim['stats']['is_unlikely'] = str(vSim['stats']['likelihood'] > self.l_thresh)
```

Comment:
@drJAGartner what happens if we don't use ```str()``` here. can we remove the 'False' string comparison above?